README.md

```diff
@@ -0,0 +1,44 @@
+# CDP Case management plugin
+
+
+## Development
+
+Before running make sure to have the following *environment variables* defined correctly:
+
+- AWS_ACCESS_KEY_ID
+- AWS_SECRET_ACCESS_KEY
+- S3_BUCKET_NAME
+- S3_BUCKET_URL
```

Comment:
what about region?

Reply:
Oh, default region can be overridden  with the proper environment variable, need to document this